ut/s3_post_complete_action_test.cc

```diff
@@ -941,14 +945,11 @@ TEST_F(S3PostCompleteActionTest, StartCleanupAbortedSinceValidationFailed) {
   EXPECT_CALL(*(object_mp_meta_factory->mock_object_mp_metadata), get_pvid())
       .WillRepeatedly(Return(pv_id));
   EXPECT_CALL(*(motr_kvs_writer_factory->mock_motr_kvs_writer),
-              put_keyval(_, _, _, _))
-      .Times(1)
-      .WillRepeatedly(
-           Invoke(this, &S3PostCompleteActionTest::dummy_put_keyval));
+              put_keyval(_, _, _, _, _, _)).Times(0);
 
   EXPECT_CALL(*(motr_writer_factory->mock_motr_writer),
               delete_object(_, _, _, _, _))
-      .Times(1)
+      .Times(0)
```

Comment:
Why is it now called 0 times and not 1 time?

Reply:
because the S3PostCompleteActionState is set to abortedSinceValidationFailed in this UT and the function call gets returned without calling the delete_object() function.